index.html

```diff
@@ -13,15 +13,20 @@
             <nav class="navigation">
                 <a href="/index.html" id="logo">Welcome</a>
     
-                <ul class="nav-list">
+                <ul class="nav-list toggle-display">
+                    <li>
+                      <button type="button" id="close-btn">
+                        <span>X</span>
+                      </button>
+                    </li>
```

Comment:
- [ ] You did a great job on the design but for this project, you are supposed to stick to [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design as much as possible. Please make sure the close(X) is on the right side of the screen instead of the left side.

### Your design
![image](https://user-images.githubusercontent.com/83803414/188502004-794cf304-22fb-4431-9ff3-eb1e036162ed.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/188502028-aa452fa3-4c92-48ba-86b8-9c73782ac735.png)